.github/workflows/ami-build-ubuntu-18.yml

```diff
@@ -80,11 +82,12 @@ jobs:
             ubuntu_release_no=${{ matrix.ubuntu_version }}
             postgresql_major=${{ steps.version.outputs.postgresql_major }}
             postgresql_release=${{ steps.version.outputs.postgresql_release }}
-            CPPFLAGS=-mcpu=${{ matrix.mcpu }}
+            CPPFLAGS=
+            DEB_BUILD_PROFILES=pkg.postgresql.nozstd
```

Comment:
Building Postgres from source requires libzstd 1.4.0+, whereas Ubuntu 18 only has versions up to 1.3.3 available.
Sadness all around, although it doesn't affect us since these binaries are only used to upgrade from PG13/14 (which didn't have zstd compression support)